cob_bringup/tools/teleop.launch

```diff
@@ -3,12 +3,15 @@
 
 	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
-	<arg name="version" default="v2"/>
 
-	<include file="$(find cob_bringup)/tools/teleop_$(arg version).xml">
+	<include file="$(find cob_bringup)/drivers/joy.launch">
 		<arg name="robot" value="$(arg robot)" />
 		<arg name="pkg_hardware_config" value="$(arg pkg_hardware_config)" />
 	</include>
 
+	<node pkg="cob_teleop" type="cob_teleop" name="$(anon teleop_node)" ns="teleop" output="screen">
+		<rosparam command="load" ns="/teleop" file="$(arg pkg_hardware_config)/$(arg robot)/config/teleop.yaml"/>
```

Comment:
What kind of parameter should this be?
private? as you put it within the `node`-tag?
or global? as you use `ns=/teleop`?
I guess the roscpp calls `getParam()` in the node need to be used with the correct NodeHandle to resolve this!